src/views/studio/lib/studio-member-actions.js

```diff
@@ -8,13 +8,20 @@ import {selectStudioId, setRoles} from '../../../redux/studio';
 const Errors = keyMirror({
     NETWORK: null,
     SERVER: null,
-    PERMISSION: null
+    PERMISSION: null,
+    DUPLICATE: null
 });
 
 const normalizeError = (err, body, res) => {
     if (err) return Errors.NETWORK;
     if (res.statusCode === 401 || res.statusCode === 403) return Errors.PERMISSION;
     if (res.statusCode !== 200) return Errors.SERVER;
```

Comment:
Errors for rate-limit (I hope it's 429?) should be handled as well, since people who try to add all followers to a studio can easily encounter that.